client/__test__/actions/signinAction.test.jsx

```diff
@@ -2,26 +2,29 @@ import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import moxios from 'moxios';
 import expect from 'expect';
-import Login from '../../actions/signinAction';
+import Login, { logout } from '../../actions/signinAction';
 import * as types from '../../actions/types';
 import mockLocalStorage from '../../__mocks__/mockLocalStorage';
+import mockSessionStorage from '../../__mocks__/mockSessionStorage';
 
 const middlewares = [thunk];
 const mockStore = configureMockStore(middlewares);
 window.localStorage = mockLocalStorage;
+window.sessionStorage = mockSessionStorage;
 
 describe('Sign in action', () => {
   beforeEach(() => moxios.install());
   afterEach(() => moxios.uninstall());
+  const Materialize = { toast: '' };
 
-  it('creates an action SET_CURRENT_USER on successful user sign up', (done) => {
+  it('dispatches an action SET_CURRENT_USER on successful user sign up', (done) => {
     moxios.stubRequest('/api/user/signup', {
       status: 201,
       response: {
         success: true,
         message: 'Sign in succesful.',
         data: {
-          token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjcsInVzZXJFbWFpbCI6ImNodWtzLm9waWFAYW5kZWxhLmNvbW0iLCJ1c2VyVXNlcm5hbWUiOiJjaHVrc3MiLCJ1c2VyUGhvbmUiOiIyMzQ3MDMzMTMwNDQwIiwiaWF0IjoxNTAzMDc1MDgwLCJleHAiOjE1MDMxNjE0ODB9.0SX6NVMqqQpgdUebW3iRBJz8oerTtfzYUm4ADESM7fk'
+          token: '0SX6NVMqqQpgdUebW3iRBJz8oerTtfzYUm4ADESM7fk'
```

Comment:
Is this a real token or mock token?
Because if it's real then it shouldn't be here. You can generate a token within your application and use here.